src/util/Permissions.js

```diff
@@ -144,6 +145,7 @@ Permissions.FLAGS = {
   USE_EXTERNAL_STICKERS: 1n << 37n,
   SEND_MESSAGES_IN_THREADS: 1n << 38n,
   START_EMBEDDED_ACTIVITIES: 1n << 39n,
+  MODERATE_MEMBERS: 1n << 40n,
```

Comment:
If its not added to -types yet, can you please add it? 🙏

Reply:
See https://github.com/discordjs/discord-api-types/pull/235!